suites/002-file-system/delete.robot

```diff
@@ -0,0 +1,67 @@
+*** Comments ***
+Copyright (c) 2025 Sine Nomine Associates
+See LICENSE
+
+
+*** Settings ***
+Documentation    Read and write files
```

Comment:
Seems to be left over from the other .robot file. Maybe needs to be updated for delete tests?

Reply:
Fixed in e42e1fc, thanks.